src/EnergyPlus/CondenserLoopTowers.hh

```diff
@@ -219,16 +219,16 @@ namespace CondenserLoopTowers {
         //  through the tower sump
         // multi cell tower
         int NumCell = 0; // Number of cells in the cooling tower
-        CellCtrl CellCtrl_Num = CellCtrl::Invalid;
+        CellCtrl CellCtrl_Num = CellCtrl::MaxCell;
```

Comment:
Changed struct `CellCtrl_Num` member variable to the default value of `CellCtrl::MaxCell`.